internal/cli/cli.go

```diff
@@ -38,3 +41,17 @@ func read(path string) ([]byte, error) {
 	}
 	return os.ReadFile(path)
 }
+
+func defaultProject(ctx context.Context) (string, error) {
+	p := project.FromApplicationDefault(ctx)
```

Comment:
That's not needed since every google sdk already does this exact thing by default... you just need a cli param to override this (default param should be empty for auto-discovery)

Reply:
@vic3lord it looks we need to set the project for the GetTrace call anyway
<img width="862" alt="image" src="https://user-images.githubusercontent.com/1474089/180453642-5850b767-0bf2-4d50-9fe9-8f70f461464e.png">